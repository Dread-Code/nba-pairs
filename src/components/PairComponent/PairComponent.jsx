import React from 'react'

const PairComponent = () => {
    return (
        <div>
            <h1>Pair Component</h1>
        </div>
    )
}

export default PairComponent
