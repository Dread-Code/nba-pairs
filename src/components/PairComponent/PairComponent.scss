.paircomponent-container {
  padding: 3vw;
}

.paircomonent-title {
  font-size: 4vh;
  text-align: center;
}
