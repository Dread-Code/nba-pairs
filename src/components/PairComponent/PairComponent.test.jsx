import React from 'react'
import { shallow } from 'enzyme'
import PairComponent from './PairComponent';

describe('PairComponent', () => {
    test('1)  should match sbnapshoot', () => {
        const wrapper = shallow(<PairComponent/>)
        expect(wrapper).toMatchSnapshot()
    });
});