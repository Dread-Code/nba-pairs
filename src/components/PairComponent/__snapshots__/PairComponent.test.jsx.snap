// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PairComponent 1)  should match sbnapshoot 1`] = `
<div>
  <h1>
    Pair Component
  </h1>
</div>
`;
